.changeset/cyan-rabbits-joke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@mysten/dapp-kit': minor
+---
+
+Have useSignPersonalMessage fall back to use sui:signMessage
```

Comment:
Do we have a `useSignMessage` hook at all?

Reply:
nah I never added one. at the time I was hopeful that wallet builders would migrate over and we'd remove `signMessage` from the standard and wallet creators, but that was probably an un-realistic thought haha

Reply:
Nah I think that's the right call, having one hook is better anyway, we can just insulate folks from this change.